packages/core/test/navbar/navbarTests.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { assert } from "chai";
+import { mount, shallow } from "enzyme";
+import * as React from "react";
+
+import { Classes, Navbar } from "../../src/index";
+
+describe("<Navbar>", () => {
```

Comment:
tautological tests, please remove

Reply:
really? all the suites? I mean yeah they're simple tests for a simple component but we need some amount of test coverage here.

Reply:
@giladgray @adidahiya - yeah, i personally prefer at least basic tests for the entire surface area of every component, just so some dunce (read: me in the future) doesn't come in and accidentally delete functionality in the future.

having untested assumptions—however small—in many places makes me nervous.

Reply:
no, I'm fairly against this. they are simple, isolated components that just render a template.

why would you come along and accidentally delete functionality? we code review everything pretty thoroughly anyway.

Reply:
but we also require a minimum 80% code coverage, and this is pretty much the only way to get that for trivial components.

possibly we could write just _one_ `Navbar` test that exercises all the sub-components and that'd be enough for karma to be quiet.

Reply:
I think this is a case where coverage is meaningless in practice. There's no tricky code here to test. Coverage shouldn't be a goal, we can exempt simple components from code coverage using `/* istanbul ignore next */`

Reply:
> why would you come along and accidentally delete functionality? we code review everything pretty thoroughly anyway.

Larger refactors, moving files, etc. I'd rather have automated checks for even the simplest things than require cognitive load from a human to give repeated stamps of approval. But I'll go ahead and remove.